Source/ChocolateyGui.Shared/Models/PackageOperationResult.cs

```diff
@@ -5,21 +5,25 @@
 // --------------------------------------------------------------------------------------------------------------------
 
 using System;
-using System.Collections.Generic;
+using System.Runtime.Serialization;
 
-namespace ChocolateyGui.Subprocess.Models
+namespace ChocolateyGui.Models
 {
+    [DataContract]
     public class PackageOperationResult
     {
         internal static readonly PackageOperationResult SuccessfulCached = new PackageOperationResult
         {
             Successful = true
         };
 
+        [DataMember]
         public bool Successful { get; set; }
 
-        public IReadOnlyList<string> Messages { get; set; } = new string[0];
+        [DataMember]
+        public string[] Messages { get; set; } = new string[0];

```

Comment:
I honestly don't know haha. It was yelling at me for other things. Will try again.